Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -481,6 +481,9 @@
         reagents:
         - ReagentId: Nutriment
           Quantity: 5
+  - type: SliceableFood
+    count: 3
+    slice: FoodTofuSlice
```

Comment:
Did you make sure to also make an item prototype for the slice? A slice is its own item with its own prototype. I can just PR the prototype of that as well as the sprites.